docs/user-guide/settings.md

```diff
@@ -0,0 +1,30 @@
+# EDA Application Settings
+
+
+Here is an example of how to configure settings for the EDA application via the EDA custom resource. This will override the default application settings if set.
+
+```yaml
+---
+apiVersion: eda.ansible.com/v1alpha1
+kind: EDA
+metadata:
+  name: eda
+spec:
+  ...
+  extra_settings:
+    - setting: MAX_RUNNING_ACTIVATIONS
+      value: "12"
+    - setting: GIT_SSL_NO_VERIFY
+      value: "true"
```

Comment:
```suggestion
```
AFAIK, passing `GIT_SSL_NO_VERIFY` through environment variables does not take any effect. Any environment variables for worker pod will not be inherited for `git` command, since `subprocess.run()` is invoked with explicit `env` param with minial dict: https://github.com/ansible/eda-server/blob/7e956b8810987fce20de698ad05a7f3e3acc323f/src/aap_eda/services/project/git.py#L192

So this two lines may cause users to be confusing. I want to suggest you to remove these lines.

Reply:
Thank you @kurokobo ! I'll remove it from the examples.